src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -75,68 +60,38 @@ public void i_click_open_SED_accordion() throws Throwable {
+	listingDetails_page.sed_details_accordion_open(driver);
+	assertTrue(driver.getPageSource().contains("Date SED Testing was Completed"));
 	}
 
-	
-
-@Then("^the download button title should read as 'Download SED Details'$")
-public void the_download_button_title_should_read_as_Download_SED_Details() throws Throwable {
+	@Then("^the download button title should read as 'Download SED Details'$")
+	public void the_download_button_title_should_read_as_Download_SED_Details() throws Throwable {
 		
-	driver.get("https://chpl.ahrqstg.org/#/product/8481");	
-	WebDriverWait wait = new WebDriverWait(driver, 20);
-	wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")));
-	
-	// to close certification criteria accordion that's open by default
-	WebElement expanded = driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)"));
-	expanded.click();
-	//to open SED accordion			
-	WebElement expanded1 = driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)"));
-	expanded1.click();
+	listingDetails_page.sed_details_accordion_open(driver);
+	assertTrue(driver.getPageSource().contains("Download SED Details"));
+	}
 	
-		assertTrue(
-				driver.getPageSource().contains("Download SED Details"));
-}
+	@Then("^usability report text should read as 'Full Usability Report'$")
+	public void usability_report_title_reads_as_Full_Usability_Report() throws Throwable {
+		listingDetails_page.sed_details_accordion_open(driver);
+		assertTrue(driver.getPageSource().contains("Full Usability Report"));
+	}
 	
-
 	@Then("^the criteria number should also include the criteria title$")
 	public void the_criteria_number_should_also_include_the_criteria_title() throws Throwable {
-		driver.get("https://chpl.ahrqstg.org/#/product/8481");
-		WebDriverWait wait = new WebDriverWait(driver, 20);
-		wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")));
+		listingDetails_page.sed_details_accordion_open(driver);
 		
-		WebElement expanded = driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)"));
-		expanded.click();
-					
-		WebElement expanded1 = driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)"));
-		expanded1.click();
-		
-		 WebElement Table_1 = driver.findElement(By.xpath("//*[@id='mainContent']/div[2]/div[2]/div/div/div[3]/div[2]/div/span/span[4]/table/thead/tr/th[1]"));
-		 List<WebElement> Rows = Table_1.findElements(By.tagName("tr"));
-		 System.out.println("No. of rows: "+Rows.size());
-		 
-		  
+		WebElement Table_1 = listingDetails_page.SED_criteria_table(driver);
+		List<WebElement> Rows = Table_1.findElements(By.tagName("tr"));
+		System.out.println("No. of rows: "+Rows.size());
+				  
 		 //to print all the values inside the table
```

Comment:
Why print all the rows? The method is looking for criteria title/number; what does that have to do with `System.out`-ing a lot of lines?